bukuserver/server.py

```diff
@@ -109,6 +110,8 @@ def create_app(db_file=None):
     app.app_context().push()
     setattr(flask.g, 'bukudb', bukudb)
     init_locale(app)
+    app.config["WTF_CSRF_ENABLED"] = False
```

Comment:
Don't turn off security in production code.

If this package doesn't work with WTF_CSRF, you can use other means for data validation (like [JSON Schema](https://pypi.org/project/jsonschema)).